scripts/plots/cam/exevs_cam_grid2obs_plots.sh

```diff
@@ -87,12 +87,13 @@ if [ -d $log_dir ]; then
     fi
 fi
 
-# Copy files to desired location
-#all commands to copy output files into the correct EVS COMOUT directory
+# Tar and Copy output files to EVS COMOUT directory
+   cd ${DATA}/${VERIF_CASE}
+   find ./* -name "*.png" -type f -print | tar -cvf ${DATA}/${NET}.${STEP}.${COMPONENT}.${RUN}.${VERIF_CASE}.v${VDATE}.tar -T -
+
 if [ $SENDCOM = YES ]; then
-    find ${DATA}/${VERIF_CASE}/out/*/*/*.png -type f -print | tar -cvf ${DATA}/${NET}.${STEP}.${COMPONENT}.${RUN}.${VERIF_CASE}.v${VDATE}.tar -T -
-    cpreq ${DATA}/${NET}.${STEP}.${COMPONENT}.${RUN}.${VERIF_CASE}.v${VDATE}.tar ${COMOUTplots}/.
-    if [ $SENDDBN = YES ]; then
-        $DBNROOT/bin/dbn_alert MODEL EVS_RZDM $job ${COMOUTplots}/${NET}.${STEP}.${COMPONENT}.${RUN}.${VERIF_CASE}.v${VDATE}.tar
-    fi
+    cpreq -v ${DATA}/${NET}.${STEP}.${COMPONENT}.${RUN}.${VERIF_CASE}.v${VDATE}.tar ${COMOUTplots}/.
```

Comment:
@RoshanShrestha-NOAA @MarcelCaron-NOAA Will lines line `cpreq -v ${DATA}/${NET}.${STEP}.${COMPONENT}.${RUN}.${VERIF_CASE}.v${VDATE}.tar ${COMOUTplots}/.` still work if you have already cd-ed into ${DATA}/${VERIF_CASE} a few lines above? 

Same question applied in all other scripts that include the new "cd ${DATA}/${VERIF_CASE}" line. Does the path to the .tar file need to change?

Reply:
Hi @AliciaBentley-NOAA 
The tar files are written to ${DATA}, so, cd ${DATA}/${VERIF_CASE} does not affect the cpreq -v command.